src/renderer/components/proxy-settings/proxy-settings.js

```diff
@@ -64,6 +63,24 @@ export default defineComponent({
     },
     proxyUrl: function () {
       return `${this.proxyProtocol}://${this.proxyHostname}:${this.proxyPort}`
+    },
+    lang: function() {
+      return this.$i18n.locale.replace('_', '-')
+    },
+    localeToUse: function() {
+      // locales found here: https://ipwhois.io/documentation
+      const supportedLangs = ['en', 'ru', 'de', 'es', 'pt-BR', 'fr', 'zh-CN', 'ja']
+      return supportedLangs.find(lang => {
+        return (this.lang === lang) || (this.lang.substring(0, 2) === lang.substring(0, 2))
```

Comment:
**nitpick:** we will have to split up this logic into two consecutive `find`s  if they ever end up adding support two locales with the same language code.

Aside from that, looks good.